install_night.html

```diff
@@ -36,15 +36,43 @@
       <!-- Any section element inside of this container is displayed as a slide -->
       <div class="slides">
         <!-- Opening slide -->
+        <section data-background-iframe="https://developer.apple.com/xcode/downloads/">
+          <h2>Installing Xcode</h2>
+          <p>Head on over to: https://developer.apple.com/xcode/downloads/</p>
+          <img style="max-width:370px; min-width:330px;" src="https://devimages.apple.com.edgekey.net/xcode/images/downloads_2x.png">
+        </section>
         <section>
-          <h3>Welcome!</h3>
-          <p>Jeff, this is you buddy :) Setup Instructions for Install Night go here.</p>
+          <h2>CocoaPods</h2>
+          <ul>
+              <li>Manages library dependencies for Xcode projects</li>
+              <li>Dependencies are specified in a single text file called a Podfile</li>
+              <li>Resolves dependencies between libraries, fetches the required source code and links everything together in an Xcode workspace to build your project</li>
+          </ul>
+          <p>Reference: </p> <a href="https://guides.cocoapods.org/using/getting-started.html">https://guides.cocoapods.org/using/getting-started.html</a>
+
         </section>
         <section>
-          <p>Second Slide</p>
+          <h2>Installing CocoaPods</h2>
+          <ol>
+              <li>Open Terminal and enter:</li>
+              <blockquote style="max-width:700px; min-width:700px;">sudo gem install cocoapods</blockquote>
+          </ol>
+          <p>NOTE: CocoaPods is built with Ruby and it will be installable with the default Ruby available on OS X</p>
         </section>
         <section>
-          <p>Third Slide</p>
+          <h2>Using CocoaPods</h2>
+          <ol>
+              <li>Navigate to your project's folder, create a file called Podfile and open it in any editor</li>
```

Comment:
We can also use TextEdit?  whatever you think works, I don't mind
introducing a new text editor!!

On Tuesday, May 12, 2015, Stephanie <notifications@github.com> wrote:

> In install_night.html
> <https://github.com/gdiboston/gdi-intro-ios/pull/1#discussion_r30191708>:
>
> >          </section>
> >          <section>
> > -          <p>Third Slide</p>
> > +          <h2>Using CocoaPods</h2>
> > +          <ol>
> > +              <li>Navigate to your project's folder, create a file called Podfile and open it in any editor</li>
>
> Good point. I don't know if they'll be comfortable with the terminal or
> vim but we can walk them through the it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdiboston/gdi-intro-ios/pull/1/files#r30191708>.
>


-- 
Best Regards,
Jeff O'Leary


[image: MeYou Health][image: FB] <https://www.facebook.com/meyouhealth>[image:
T] <http://twitter.com/#!/meyouhealth>[image: RSS]
<http://www.meyouhealth.com/blog/>[image: YT]
<http://www.youtube.com/user/meyouhealthvideos>[image: F]
<http://www.flickr.com/photos/meyouhealth>
*Jeff O'Leary*iOS Engineer
Cell: 857-268-2939
MeYou Health - the social well-being companywww.meyouhealth.com
jeff.oleary@meyouhealth.com
MeYou Health is a Healthways, Inc. company